persistence/src/main/java/com/bionicuniversity/edu/fashiontips/entity/Post.java

```diff
@@ -103,6 +102,11 @@
     private boolean commentsAllowed = true;
 
     /*
+     * Enable notification about new comment
+     * */
+    @Column(name = "notification", nullable = false)
```

Comment:
@barik111 I recommend you to change the field name, for instance to "notification_enabled". Now it looks like the field describes IS THE POST - NOTIFICATION?